app/components/personas/view.html.erb

```diff
@@ -0,0 +1,19 @@
+<hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+
+<h2 class="govuk-heading-l"><%= persona.name %></h2>
+
+<% if roles.any? %>
+  <div class="govuk-!-margin-bottom-5">
+    <% roles.each do |role| %>
+      <%= render_component GovukComponent::Tag.new(text: role, colour: "blue") %>
+    <% end %>
+  </div>
+<% end %>
```

Comment:
dead end?

Reply:
I was wondering the same thing. Roles might come down the line, but I would be inclined to remove it until a future requirement demands it.

Reply:
Yeah. We don't have roles as yet. Let's get rid.

Reply:
Removed